includes/event/event.php

```diff
@@ -59,8 +59,8 @@ public function __construct(
 		string $status,
 		string $title,
 		string $description,
-		DatetimeImmutable $created_at = null,
-		DatetimeImmutable $updated_at = null,
+		DateTimeImmutable $created_at = null,
+		DateTimeImmutable $updated_at = null,
```

Comment:
I'm a bit confused by `$created_at` and `$updated_at`, they don't seem to be used anywhere, should we just remove them?

Reply:
Thank you for reviewing! These properties were introduced in the [RSS PR](https://github.com/WordPress/wporg-gp-translation-events/pull/326/) and `$updated_at` was used [here](https://github.com/WordPress/wporg-gp-translation-events/blob/4eccfd26a3a42bd3fc68b1b07ab45faee0721cf5/includes/routes/event/rss.php#L105).

Perhaps, it's `$created_at` that may be removed.

Reply:
Ah indeed, I missed that, I only searched for `created_at`.